subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/PreTransportMappers.kt

```diff
@@ -20,12 +22,15 @@ internal interface PreTransportMappers {
     fun DataSet.serialize(): Map<String, String> {
         return javaClass.declaredFields
             .filter { it.name != "number" }
-            .map { field ->
-                field.name to javaClass.methods
-                    .find { it.name == "get${field.name.capitalize()}" }
-                    ?.invoke(this)
-                    .toString()
-            }
+            .filter { !it.type.isLambda() }
+            .map { field -> field.name to getFieldStringValue(field) }
```

Comment:
Not sure, but can't we use kotlin property https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.reflect.jvm/java.lang.reflect.-field/ instead, can we?

Reply:
I think we could, not sure if java supported at all now